CIP-meta-assets (ERC20-like assets)/README.md

```diff
@@ -0,0 +1,448 @@
+
+#### `Transfer`
+
+The `Transfer` redeemer is used to pay an other account in the same account manager.
+
+When used as redeemer the contract checks for the following conditions to be true;
+
+- `ownInputs` to be of lenght `2`
+
+- the receiver input (the `ownInput` with different `utxoRef` of the `validatingInput`)
+being spent with `Receive` redeemer
+
+- the sender input (`validatinInput`) must have the NFT according to `account.currencySym` (the input is valid)
+
+- the output with the sender's credentials must preserve the NFT
+
+- the account fields `credentials`, `currencySym` and `state` are unchanged between sender input datum and sender output datum
+
+- the account fields `credentials`, `currencySym` and `state` are unchanged between receiver input datum and receiver output datum
+
+- `ownOuts` to be of lenght `2`
+
+- every member of `ownOuts` must have at most 2 entries to prevent DoS by token spam.
+
+- the input sender's amount field is greather or equal than the redeemer amount (`transfer.amount`)
+
+- the output sender's amount field is equal to the input one minus `transfer.amount`
+
+- the output receiver's amount field is equal to the input one plus `transfer.amount`
+
+- the sender singned the transaction (included in `ctx.tx.signatories` if a `PPubKeyHash` or included a script input if a `PValidatorHash`)
+
+any additional check can be made based on the `account.state` (implementation specific)
+
+#### `Receive`
+
+- `ownInputs` to be of lenght `2`
+
+- the sender input (the `ownInput` with different `utxoRef` of the `validatingInput`)
+being spent with `Transfer` redeemer
+
+- in the sender input `Transefer` redeemer the `to` field is equal to the `account.credentials` (where `account` is the receiver datum)
+
+> **NOTE**
+>
+> the sender input being an `ownInputs` element and being spent with `Transfer` redeemer 
+> implies that the transfer calculation is running in that validation
+>
+> hence we don't need to perform the same calculation here
+
+#### `ForwardCompatibility`
+
+For the current version this redeemer SHOULD always fail.
+
+The main purpose of the redeemer is to avoid breaking compatibilities for addtional implementation specific redeemers
+
+## Rationale: how does this CIP achieve its goals?
+<!-- The rationale fleshes out the specification by describing what motivated the design and what led to particular design decisions. It should describe alternate designs considered and related work. The rationale should provide evidence of consensus within the community and discuss significant objections or concerns raised during the discussion.
+
+It must also explain how the proposal affects the backward compatibility of existing solutions when applicable. If the proposal responds to a CPS, the 'Rationale' section should explain how it addresses the CPS, and answer any questions that the CPS poses for potential solutions.
+-->
```

Comment:
```suggestion
```
**No** you don't need the comment but **yes** please contribute a section as indicated in this comments.  You could paraphrase a bunch of reviewer comments & your own responses already in this PR and have a decent Rationale for this proposal... at least to start with.  In any case it _does_ need _something_.

Reply:
Please do add applicable content here @michele-nuzzi, this is a requirement for us to be able to assign a number and then merge.